hardhat.config.ts

```diff
@@ -35,9 +35,7 @@ import {
   withRetries,
 } from './src/network';
 
-const THEGRAPHURLS: { [key: string]: string } = {
-  goerli: 'https://api.thegraph.com/subgraphs/name/balancer-labs/balancer-authorizer-goerli',
-};
+const THEGRAPHURLS: { [key: string]: string } = {};
```

Comment:
Do we need this at all? Only seems to be used in the `get-action-ids-info` task. Do we need that?